src/NTH.Windows.Forms/NTH.Windows.Forms/PlaceholderTextbox.cs

```diff
@@ -33,9 +33,14 @@ public PlaceholderTextbox()
 
         private void UpdatePlaceholder()
         {
-            var currentHandle = Handle;
-            if (currentHandle != IntPtr.Zero)
-                NativeMethods.SendMessage(currentHandle, WindowMessage.EM_SETCUEBANNER, IntPtr.Zero, _placeholder);
+            if (IsHandleCreated)
+                NativeMethods.SendMessage(Handle, WindowMessage.EM_SETCUEBANNER, new IntPtr(1), _placeholder);
```

Comment:
So, maybe we can add a property like this:
```C#
private bool _remainPlacerholderOnFocus;
public bool RemainPlacerholderOnFocus // or something similar
{
	get
	{
		return _remainPlacerholderOnFocus;
	}
	set
	{
		if(_remainPlacerholderOnFocus != value)
		{
			_remainPlacerholderOnFocus = value;
			UpdatePlaceholder();
		}
	}
}

// ...
NativeMethods.SendMessage(Handle, WindowMessage.EM_SETCUEBANNER, new IntPtr(_remainPlacerholderOnFocus ? 1 : 0), _placeholder)
```

Since this library will be published via NuGet, not every developer references the source code of this library directly and will be able to change this behavior (if he can even find the line of code responsible for this behavior).